app/controllers/styles_controller.rb

```diff
@@ -3,4 +3,11 @@ class StylesController < ApplicationController
    def index
       @styles = Style.all
    end
+
+   def show
+      @style = Style.find(params[:id])
+      return @images if @images
+      image_service = GoogleSearchResults.new(q: "tattoo #{ @style.name }", tbm: "isch", serp_api_key: ENV['SERP_API_KEY'])
+      @images = image_service.get_hash[:images_results][0..29]
+   end
```

Comment:
On the `user_registration` branch, I created a few poros to handle these types of asks. This will be an easy spot to refactor to move things out of the controller once my branch is merged in :)